### DISCUSSION TITLE
Filtering cases by Cause

### DISCUSSION THREAD

**Discussion by Eran-T** [freelawproject/courtlistener#3291] [Q&A] [answered]
Hi,
I was wondering if there is a way to filter cases by the cause field. If so which endpoints support this type of operation? 

Thank you!

**Comment by mlissner** [comment, marked as answer]
Unfortunately no, this isn't currently possible. What I'd advise instead is to grab the data from the FJC IDB and use it as a starting point. It often has the cause field populated, and if you start there, you can then filter our data. 

The cause field is difficult because it doesn't appear anywhere that we can easily parse. It might be tucked into some of the PDFs — I haven't checked, but I suspect so — but if that's the case, it's not something we've tried to pull out yet. 

That leaves the FJC data, which is pretty darned good. If you end up working with it, we have a page with some advice here: 

https://free.law/idb-facts/

**Reply by Eran-T** [reply]
Thanks!
Do you happen to know if the FJC covers the entirety of Pacer?

**Reply by mlissner** [reply]
It does, yes. That's why it's so great.

**Reply by mlissner** [reply]
OK, I have to confess I remembered this entirely wrong, and somebody else on my team sent me a message asking what I was talking about. The cause field is _indeed_ on the docket sheet in PACER, and we do _indeed_ capture it on the docket object, so when it's available, we get it. That's the good news! The bad news is that we don't have a database index on the column, and we don't make our content filterable by it. 

So I think the answer is mostly the same: Use FJC, but apologies for getting the details of our system wrong. Totally my mistake.

**Reply by Eran-T** [reply]
Thanks for the update!
